cloud_edition/flexibility_mode/docs/environments_access.rst

```diff
@@ -138,3 +138,83 @@ Copy data from one instance to another
     On the SCP command, please note that no domain is specified.
 
     Use the short host name of instance. Connect to the target server and run `hostname` to get this value.
+    
+    
+Copy process (MySQL data & pictures (including assets)) from one instance to another
+************************************************************************************
+
+**Scenario:**
+
+    User wants to copy all the data (including assets & pictures) from staging instance to production instance :
+
+**Step one: Import MySQL data**
+   
+    First, create a MySQL dump of your data:
+
+.. code-block:: bash
+
+    #on mystaging.cloud.akeneo.com
+    mysqldump -u akeneo_pim -p$APP_DATABASE_PASSWORD akeneo_pim > /home/akeneo/pim/dump_dd_mm_yy.sql
```

Comment:
```suggestion
    mysqldump -u akeneo_pim -p$APP_DATABASE_PASSWORD akeneo_pim > /home/akeneo/pim/dump_`date -u +"%Y-%m-%dT%TZ"`.sql
```
That will interpolate the UTC date & time, so the file will have a name like `dump_2022-11-23T17:28:05Z.sql`